projects/storefrontstyles/scss/components/product/list/_product-list.scss

```diff
@@ -63,6 +63,10 @@
     margin: auto;
   }
 
+  .cx-single-btn-container div.btn.btn-block.btn-action {
```

Comment:
good good but lets lessen the calls and just put .btn-block or action 
this will covber both back to top and show more

Reply:
move the second part above to the .cx-single-btn-container